language-server/modules/langserver-core/src/test/resources/completion/function_body/source/source16.bal

```diff
@@ -0,0 +1,15 @@
+import ballerina/module1;
+
+function demo() {
+}
+
+service / on new module1
+# This returns list of services in all namespaces.
+# + return - list of services in namespaces.
```

Comment:
Add a newline to separate constructs.
Also, notice no period for return parameter documentation.

```suggestion
service / on new module1

# This returns a list of services in all namespaces.
# + return - list of services in namespaces
```